mix.exs

```diff
@@ -51,13 +51,14 @@ defmodule Beaker.Mixfile do
 
   defp deps do
     [
-      {:phoenix, "~> 1.1", optional: true},
+      {:earmark, "~> 0.1", only: :docs},
       {:ecto, "~> 1.1", optional: true},
+      {:ex_doc, "~> 0.11", only: :docs},
+      {:inch_ex, "~> 0.4.0"},
```

Comment:
Quite right!  It doesn't need to be there.  Was adding the version because Mix was complaining and missed the environment.  Will fix.